src/constants/typography.ts

```diff
@@ -5,26 +5,25 @@ const typography = {
   primary: `"Open Sans",${systemFontStack}`,
   sizes: {
     text: {
-      base: '16px',
-      small: '14px',
+      lead: '18px',
+      body: '15px',
+      small: '13px',
     },
     heading: {
+      display: '68px',
       h1: '48px',
-      h2: '28px',
-      h3: '24px',
-      h4: '20px',
-      h5: '20px',
-      h6: '20px',
+      h2: '38px',
+      h3: '28px',
+      h4: '24px',
+      h5: '18px',
+      h6: '15px',
     },
   },
   weights: {
     regular: 400,
-    semibold: 600,
+    semiBold: 600,
     bold: 700,
-  },
-  lineHeights: {
-    text: 1.6,
-    heading: 1.1,
+    extraBold: 800,
```

Comment:
extraBold is not a thing. The maximum weight we have is 700.

Reply:
It's definitively a thing in web and we had a whole meeting about it on Friday 😁

What are you following for the source of truth on this? 

We're following what's on [Marvel](https://marvelapp.com/5504e6j/screen/67099627). and catching up with design daily 👍🏻